src/magic/model/event/MagicSurveilEvent.java

```diff
@@ -0,0 +1,47 @@
+package magic.model.event;
+
+import magic.model.MagicGame;
+import magic.model.MagicPlayer;
+import magic.model.MagicSource;
+import magic.model.action.SurveilAction;
+import magic.model.choice.MagicSurveilChoice;
+import magic.model.trigger.MagicTriggerType;
+
+public class MagicSurveilEvent extends MagicEvent {
+
+    public MagicSurveilEvent(final MagicEvent event) {
+        this(event.getSource(), event.getPlayer(), true);
+    }
+
+    public MagicSurveilEvent(final MagicSource source, final MagicPlayer player) {
+        this(source, player, true);
+    }
+
+    private MagicSurveilEvent(final MagicSource source, final MagicPlayer player, final boolean trigger) {
+        super(
+                source,
+                player,
+                new MagicSurveilChoice(),
+                trigger ? 1 : 0,
```

Comment:
surveil should also have trigger be true right? If so, trigger should be removed.

Reply:
there is no pseudo-surveil action like it was with scry, so no need to make trigger conditional. Fixed.